tests/CommandsTest.php

```diff
@@ -116,8 +118,19 @@
     expect(Schema::hasTable('users'))->toBeFalse();
 });
 
-// Incomplete test
-test('seed command works');
+test('seed command works', function (){
+    $tenant = Tenant::create();
+    Artisan::call('tenants:migrate');
+    expect(Schema::hasTable('users'))->toBeFalse();
+
+    Artisan::call('tenants:seed', ['--class' => TestSeeder::class]);
```

Comment:
This part doesn't make sense. You're making assertions about the central table (that the `users` table doesn't exist there), but it's just confusing because **it looks as if the seeder created the table**.

Also, you're not actually asserting that the seeder does something.

You're meant to assert that there are no users in the tenant, then run the seeder, and then assert that there are users in the tenant.

Reply:
Thank you for the review. Yes, this test was not making sense, how dumb. I've just updated the test now. Now we assert user's table is empty before running the seeder command, and after running the seeder command, we assert that table got the user.